weather/module.py

```diff
@@ -18,6 +21,93 @@ class Weather(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
 
+    def _get_useful_data(self, json, ctx, lang_preference):
+        """
+        example json: https://wttr.in/praha?lang=sk&format=j1
+        get useful data from json as list of individual days
+        """
+        weather = []
+        # get individual days to extract data
+        num_of_forecast_days = 3 # number of days to get forecast for (including current day, max is 3)
+        day_phases = { # dict for getting the data from json easier (when you don't wan't some phase of day comment it)
+            "Morning": 2,
+            "Day": 4,
+            # "Evening": 6,
+            "Night": 7
+        }
+        for i in range(num_of_forecast_days):
+            cur_day = json['weather'][i]
+            day_dict = {
+                "date": cur_day['date'],
+            }
+            cur_day = cur_day['hourly']
+            for k, v in day_phases.items():
```

Comment:
This seems kinda obscure to me. I think that k and v should be renamed to something that better says what are those variables holding.

Reply:
Agreed.